templates/home/_fsf_rust.html

```diff
@@ -10,8 +10,41 @@ <h4>Why are snaps good for Rust projects?</h4>
     </ul>
   </div>
 
+  <div class='col-6'>
+    <h4>How Parity defines snapcraft.yaml:</h4>
+    <code class="p-code-yaml"><pre>
+<b>name</b>: parity
+<b>version</b>: git
+<b>summary</b>: A fast CSV command line toolkit written in Rust
+<b>description</b>: |
+  xsv is a command line program for indexing, 
+  slicing, analyzing, splitting and joining CSV 
+  files. Commands should be simple, fast and 
+  composable:
+  - Simple tasks should be easy.
+  - Performance trade offs should be exposed 
+    in the CLI interface.
+  - Composition should not come at the 
+    expense of performance.
+
+<b>confinement</b>: devmode
+<b>base</b>: core18
+
+<b>parts</b>:
+  <b>xsv</b>:
+    <b>plugin</b>: rust
+    <b>source</b>: .
+
+<b>apps</b>:
+  <b>xsv</b>:
+    <b>command</b>: bin/xsv
+    <b>plugs</b>:
```

Comment:
Don't specify plugs. We won't cover strict confinement here and they're meaningless in devmode.